src/test/java/com/epam/tc/hw2/ExerciseOneTest.java

```diff
@@ -0,0 +1,86 @@
+package com.epam.tc.hw2;
+
+import java.time.Duration;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class ExerciseOneTest extends AbstractTest {
+
+
+    private static String XPATH_ITEMS_IN_HEADER = "//ul[@class=\"uui-navigation nav navbar-nav m-l8\"]";
+    private static String XPATH_LEFT_MENU = "//*[@class='sidebar-menu left']";
+    private static String ID_FRAME_BUTTON = "frame-button";
+    private static String LIST_ITEMS_HEADER = "HOME\nCONTACT FORM\nSERVICE\nMETALS & COLORS";
```

Comment:
This text looks really specific. It's better to create locator to find 4 elements and get text from them as list (it's possible)